### PR TITLE
January Newsletter

### PR DESCRIPTION
**Pull request by davidfischer**
Please review this!

### CODE REVIEW

**Comment by treyhunner** [comment]
:+1: looks good to me

File: content/posts/2015-01-03_january-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+Title: January Newsletter
+Date: 2015-01-03
+Tags: newsletter
+Slug: january-newsletter-2015
+Author: David Fischer
+
+
+Happy New Year,
```
Comment (macro1) [inline content/posts/2015-01-03_january-newsletter.md]: Show some enthusiasm! :stuck_out_tongue_closed_eyes:

**Comment by macro1** [comment]
Looks good.

**Comment by alaindomissy** [comment]
Looks good

Plot.ly event is on Wed:

San Diego Data Science R Users
WhenWed, January 7, 6pm – 9pm
Descriptionhttp://www.meetup.com/San-Diego-Data-Science-R-Users-Group/
http://ansirsd.com/events/

**Comment by davidfischer** [comment]
There are essentially no details on the plot.ly event.

**Comment by davidfischer** [comment]
By that I mean there's no website, there's no talk of what will be presented and there's no link. I'm considering leaving it out for now.

**Comment by alaindomissy** [comment]
Hold on. Let me give you something... One sec...

**Comment by alaindomissy** [comment]
David:

Matt Sundquist co-founder at Plotly https://plot.ly will talk about data
visualization in general and how more specifically his company is thinking
about it.
Plotly is a platform for analyzing data and making interactive graphs with
Python, matplotlib, R and MATLAB.
Plotly allows 2D, 3D, and live-streaming graphs rendered with D3.js, a
JavaScript visualization library.
Plotly works with IPython.

Marc worked on the Facebook privacy team before Plotly and was a Harvard
Law School Program on the Legal Profession Student Fellow.
Marc is a writer for SCOTUSblog.com. He was a Fulbright Scholar in
Argentina, and philosophy major at Harvard.

plot.ly also offers a A JavaScript Charting Library for Scientists,
Engineers, and Quants:
https://plot.ly/product/plotlyjs/

I hope this helps?

Thanks
--Alain

**Comment by davidfischer** [comment]
Thanks

**Comment by davidfischer** [comment]
Are we 100% sure this event is happening? It isn't on the R group page and it isn't confirmed at he'll be there on the Ansir events page.

**Comment by davidfischer** [comment]
I'm just a little concerned about promoting it if it turns out to not happen.

**Comment by alaindomissy** [comment]
David, it is 100% sure to happen.
I got confirmation from Derren.
Derren had just forgotten to post it on the Data Science and R meetup page.
He said he will be posting it asap tonight.
--Alain

**Comment by alaindomissy** [comment]
David,

At the bottom of the newsletter, when we list the special workshop, could
we include the last one ?
Django One-Day Unconference
http://www.meetup.com/pythonsd/events/212859872/
What about the upcoming one:
Data Science workshop
http://www.meetup.com/pythonsd/events/219582270/


I have posted but not announce yet the plotly event:
http://www.meetup.com/pythonsdd/events/219593100/
It is now also on the SD Data Science and R user group:
http://www.meetup.com/San-Diego-Data-Science-R-Users-Group/events/219589194/
Should we include these links in the newsletter.
Should we announce the event on meetup now?

Thanks
--Alain

**Pull request merged**